mg400_plugin/src/motion_api/mov_j.cpp

```diff
@@ -104,7 +104,9 @@ void MovJ::execute(const std::shared_ptr<GoalHandle> goal_handle)
       return is_in_tolerance(pose.position.x - goal.position.x, tolerance_mm) &&
              is_in_tolerance(pose.position.y - goal.position.y, tolerance_mm) &&
              is_in_tolerance(pose.position.z - goal.position.z, tolerance_mm) &&
-             is_in_tolerance(pose.orientation.w - goal.orientation.w, tolerance_rad);
+             is_in_tolerance(
+        tf2::getYaw(pose.orientation) - tf2::getYaw(goal.orientation),
```

Comment:
It seems like `Uncrustify` requires these line to have this indentation (if not, the test fails)